packages/i18n-calypso/src/number-format.js

```diff
@@ -0,0 +1,38 @@
+/** @TODO prettier, eslint fix, test, refactor+update */
+/* eslint-disable prefer-const, no-shadow */
+
+/**
+ * Exposes number format capability
+ *
+ * @copyright Copyright (c) 2013 Kevin van Zonneveld (http://kvz.io) and Contributors (http://phpjs.org/authors).
+ * @license See CREDITS.md
+ * @see https://github.com/kvz/phpjs/blob/ffe1356af23a6f2512c84c954dd4e828e92579fa/functions/strings/number_format.js
+ */
+
+export default function number_format( number, decimals, dec_point, thousands_sep ) {
```

Comment:
This is available via npm now as part of [locutus](https://npmjs.com/package/locutus). Let's see what impact that has on the build. We might also consider pulling out this specific package in isolation if licenses allow.

At a glance, the implementation appears largely the same:

https://github.com/kvz/locutus/blob/485a9562b0435c646013bf27e84941a946ec3c97/src/php/strings/number_format.js

Reply:
Let's leave this for a follow up